codex/full/deploy.yml

```diff
@@ -0,0 +1,223 @@
+---
+version: "2.0"
+
+endpoints: 
+  nodeip2:
+    kind: ip
+
+services:
+  codex:
+    image: codexstorage/nim-codex:0.2.0
+    expose:
+    - port: 8070
+      as: 8070
+      proto: tcp
+      to:
+        - global: true
+          ip: nodeip2
+    - port: 8090
+      as: 8090
+      proto: udp
+      to:
+        - global: true
+          ip: nodeip2
+    - port: 8080
+      as: 8080
+      to:
+        - service: nginx
+    env:
+      - DATA_DIR=/codex/data
+      - ETH_PRIVATE_KEY=
+      - PUBLIC_IP=
+      - CODEX_LOG_LEVEL=debug
```

Comment:
Not sure the log_level or block_TTL need to be set here. Defaults not OK?

Reply:
:shrug: Sure